test/distribution/test_inflated_beta.py

```diff
@@ -0,0 +1,50 @@
+import mxnet as mx
```

Comment:
The header with license notice is missing, you can fix it by running

```
.devtools/license fix test
```

which should only affect the present file (if everything else within `test` has the header)